cmd/erasure-sets.go

```diff
@@ -403,6 +405,18 @@ func newErasureSets(ctx context.Context, endpoints PoolEndpoints, storageDisks [
 				if disk == nil {
 					continue
 				}
+
+				if disk.IsLocal() && globalIsDistErasure {
+					globalLocalDrivesMu.RLock()
+					ldisk := globalLocalSetDrives[poolIdx][i][j]
+					if ldisk == nil {
```

Comment:
If it's not fetched, but storageDisks[i*setDriveCount+j] is fetched inside, is this possible?

Reply:
no we are ignoring if we do not have any local drives registered, that will be populated back in connectDisks()